### ISSUE TITLE
Standard install not working 

### ISSUE THREAD

**Issue by benfrain** [p00f/nvim-ts-rainbow#65]
**Describe the bug**
Installed as per the readme but see no rainbow brackets (tried CSS/TS). Tried with both Tokyo Night and Everforest themes.

Using NVIM v0.6.0-dev+1607-g7d2233fad

:TSConfigInfo output:

```
{                                                                                                                            
  ensure_installed = {},                                                                                                     
  ignore_install = {},                                                                                                       
  modules = {                                                                                                                
    highlight = {                                                                                                            
      additional_vim_regex_highlighting = false,                                                                             
      custom_captures = {},                                                                                                  
      disable = { "markdown" },                                                                                              
      enable = true,                                                                                                         
      loaded = true,                                                                                                         
      module_path = "nvim-treesitter.highlight"                                                                              
    },                                                                                                                       
    incremental_selection = {                                                                                                
      disable = {},                                                                                                          
      enable = false,                                                                                                        
      keymaps = {                                                                                                            
        init_selection = "gnn",                                                                                              
        node_decremental = "grm",                                                                                            
        node_incremental = "grn",                                                                                            
        scope_incremental = "grc"                                                                                            
      },                                                                                                                     
      module_path = "nvim-treesitter.incremental_selection"                                                                  
    },                                                                                                                       
    indent = {                                                                                                               
      disable = {},                                                                                                          
      enable = false,                                                                                                        
      module_path = "nvim-treesitter.indent"                                                                                 
    },                                                                                                                       
    rainbow = {                                                                                                              
      colors = {},                                                                                                           
      disable = {},                                                                                                          
      enable = true,                                                                                                         
      extended_mode = true,                                                                                                  
      loaded = true,                                                                                                         
      module_path = "rainbow.internal",                                                                                      
      termcolors = {}                                                                                                        
    }                                                                                                                        
  },                                                                                                                         
  update_strategy = "lockfile"                                                                                               
}  
```

Also tried moving setup until after the colorscheme is set but makes no difference.

**Steps to reproduce**
Using this init.lua where everything else works as expected: https://gist.github.com/benfrain/97f2b91087121b2d4ba0dcc4202d252f try opening TS/CSS file

**Expected behavior**
To see rainbow brackets

**Screenshots**
![image](https://user-images.githubusercontent.com/1318466/131656594-ae483485-63e4-4910-845a-a0bcf4060d27.png)

**Comment by ram02z** [comment]
@benfrain The `colors` and `termcolors` tables are empty. You can either add your own colors (see lua/rainbow/colors.lua and lua/rainbow/termcolors.lua) or you can remove the two tables to use the default colors.

**Comment by benfrain** [comment]
@ram02z even if I set both those, I still don't see anything:

```
require("nvim-treesitter.configs").setup({
  rainbow = {
    enable = true,
    extended_mode = true, -- Also highlight non-bracket delimiters like html tags, boolean or table: lang -> boolean
    max_file_lines = nil, -- Do not enable for files with more than n lines, int
    colors = { "#cc241d", "#a89984", "#b16286", "#d79921", "#689d6a", "#d65d0e", "#458588" }, -- table of hex strings
    termcolors = { "#cc241d", "#a89984", "#b16286", "#d79921", "#689d6a", "#d65d0e", "#458588" }, -- table of colour name strings
  },
})
```

**Comment by ram02z** [comment]
@benfrain `termcolors` use xterm color names, so hex values would be incorrect. I would reccommend omitting it since termcolors are used to change the `ctermfg`, which is used when the console version of nvim is used. If you want to change the defaults, take a look at [lua/rainbow/termcolors.lua](https://github.com/p00f/nvim-ts-rainbow/blob/master/lua/rainbow/termcolors.lua).

**Comment by p00f** [comment]
> I'd assumed it would fallback to some defaults if not set

If you leave out those fields they fall back to a default (ie dont set an empty table, just skip it entirely)

**Comment by benfrain** [comment]
@ram02z great, thanks, ommitted the two maps and I'm up and running

**Comment by benfrain** [comment]
@p00f @ram02z think README only needed slight change, hope the PR helps. Thanks again for the plugin!

**Comment by p00f** [comment]
Fixed in 2f39c59
